msal/application.py

```diff
@@ -251,6 +252,10 @@ def get_authorization_request_url(
         :param nonce:
             A cryptographically random value used to mitigate replay attacks. See also
             `OIDC specs <https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest>`_.
+        :param domain_hint:
+            Provides a hint about the tenant or domain that the user should use to sign in. The value
+            of the domain hint is a registered domain for the tenant.
```

Comment:
So, what can be the proper value to be used in this feature? This line suggests "a registered domain for the tenant" which sounds something like "contoso.com" (which is probably correct, based on [the link mentioned here](https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/158#issuecomment-584890191)), but the link at the next line below suggests two (and only two?) valid values as "consumers" or "organizations". Which one is correct?

If all those can be valid values, we can then choose to give a more clear description, something like this:

> Can be one of "consumers" or "organizations" or your tenant domain "contoso.com".
> If included, it will skip the email-based discovery process that user goes through on the sign-in page, leading to a slightly more streamlined user experience.
> And then provide a link (or even two) to existing docs to back it up.

Reply:
Yes it can be contoso.com too, I changed the doc string to what you mentioned